libraries/core_libs/consensus/include/pbft/pbft_manager.hpp

```diff
@@ -56,6 +56,15 @@ class PbftManager : public std::enable_shared_from_this<PbftManager> {
   size_t getTwoTPlusOne() const;
   void setPbftStep(size_t const pbft_step);
 
+  /**
+   * @brief Generate PBFT block, push into unverified queue, and broadcast to peers
+   * @param prev_blk_hash previous PBFT block hash
+   * @param anchor_hash proposed DAG pivot block hash for finalization
+   * @param order_hash the hash of all DAG blocks include in the PBFT block
+   * @return PBFT block
+   */
+  std::shared_ptr<PbftBlock> generatePbftBlock(const blk_hash_t &prev_blk_hash, const blk_hash_t &anchor_hash,
```

Comment:
You never used result of this function, just the hash of pbft block

Reply:
Fixed